README.md

```diff
@@ -16,56 +16,101 @@ halt "rogue" pull requests before they're merged into the master branch.
+for your convenience, with which you can either start a database for development (`postgres-dev`) **or** testing: (`postgres-test`).  
+Since dev and test database share the same port, you should either change the port of one of the databases or comment out in the `docker-compose.yaml` 
+the database you currently don't need.  
+
+Make sure your docker machine host is running:
+
+~~~ shell
+ docker-machine start
+~~~  
 
+Start database and schemas:
 ~~~ shell
-# starts 2 postgres docker containers, one for test and one for development purposes
+export DB_HOST="$(docker-machine ip)"  
+
+# Starts the postgres docker container for development & test purposes
+# run either
 docker-compose up postgres-dev
-docker-compose up postgres-test
+# or
+docker-compose up postgres-test 
+
 # creates database and schemas, only needed first time
+# run it database command in additional terminal session
 ./init_db.sh
-export DB_HOST="$(docker-machine ip)"
-npm run all
+
+
 ~~~
+To get your docker-machine ip you can also run `docker-machine ip` in the shell and copy the address.  
 
-If you would like to run a complete Zappr locally, you can do it like this:
+#### Zappr Setup
+
+To get your `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` copy it from your Github `Settings/ Developer Settings/ OAuth Apps` or 
+create them new following these [instructions](https://auth0.com/docs/connections/social/github). 
+Export your [Github credentials](https://github.com/settings/applications) and docker-machine IP:
 
-1. Export your [Github credentials](https://github.com/settings/applications) and docker-machine IP:
 ~~~ shell
 export DM_IP="$(docker-machine ip)"
 export GITHUB_CLIENT_ID=<your-client-id>
 export GITHUB_CLIENT_SECRET=<your-client-secret>
 ~~~
 
-2. Start Zappr via docker compose or npm
-~~~ shell
-docker-compose up
-~~~
+Start Zappr with any of the following alternatives  
+##### Using NPM:
+ - Precondition VPN Setup
+    Install and run localtunnel to expose your localhost in a separate terminal window
+    ~~~ shell
+    npm i -g localtunnel
+    lt -s <your-app-name> -p 3000
+    ~~~
+    Double check that your Authorization callback URL is set to `https://<your-app-name>.localtunnel.me` in the GitHub settings.
 
 ~~~ shell
 npm install
 npm run build
-npm run all
+GITHUB_CLIENT_ID=<your-client-id> GITHUB_CLIENT_SECRET=<your-client-secret> HOST_ADDR=https://<your-app-name>.localtunnel.me/ npm run all
+# you may ommit GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET if exported to regarding shell environment variables earlier
 ~~~
 
-3. Install and run localtunnel to expose your localhost
-~~~ shell
-npm i -g localtunnel
-lt -s myzappr -p 3000
-~~~
+Go to `https://<your-app-name>.localtunnel.me` and do things :)
 
-4. Set your OAuth application URLs to `https://myzappr.localtunnel.me` in the GitHub settings
+##### Using docker compose:    
+  ~~~ shell
+  docker-compose up
+  ~~~
+  @TOOO: how to access the local zappr installation
```

Comment:
@lotharschulz what do you mean with how to access local zappr installation? docker-compose up should run the databases and do npm start -> therefore on localhost you should be able to access it, shouldn't it ?

Reply:
I hope so! `docker-compose up` did no work for me and I could neither find out why nor fix it, so I left the @TODO. Does it work for you?